frontend/src/components/LandingPage/Tools/Tools.js

```diff
@@ -7,7 +7,8 @@ import {
   function Tools() {
     return (
       <div>
-        <Jumbotron fluid>
+        <Jumbotron fluid style={{ background: `linear-gradient(25deg, #e3c3b2 25%, rgba(0, 0, 0, 0) 25%), 
```

Comment:
What do you think about moving this to the stylesheet? If its not applying to the jumbotron I can show you how.

Reply:
Yeah, I was not able to style it in the stylesheet when I was doing it, but I found a work around by using !important to override the default background. Let me know if this was a good/bad way to override it!